### ISSUE TITLE
Bucket creation

### ISSUE THREAD

**Issue by lericson** [lericson/simples3#1]
One should be able to create buckets. Suggested API for this is either:

```
from simples3 import S3Bucket
b = S3Bucket("foo", ...)
b.create(acl="public-read")
```

or class-style (a bit better IMO):

```
from simples3 import S3Bucket
b = S3Bucket.create_new("foo", ..., acl="public-read")
```

There are other suggestions as well, including module-level functions.

**Comment by lericson** [comment]
Fixed in 00ed9ac

**Comment by lericson** [comment]
Second attempt, lericson/simples3@00ed9ac76fd91e53bbec4f495eaa91906da8a94f
